### ISSUE TITLE
BufferUnderflowException in xyz.gianlu.librespot.core.Session$Receiver.run(Session.java:756)

### ISSUE THREAD

**Issue by pejobo** [librespot-org/librespot-java#152]
**Describe the bug**
Started the fat jar v0.6.2 and tried to connect with android app.

**To Reproduce**
Start with `java -jar librespot-core-jar-with-dependencies.jar` and try to connect with android app.

Tried on ubuntu and raspberry with same outcoume
```
$ java -version
openjdk version "11.0.2" 2018-10-16
OpenJDK Runtime Environment 18.9 (build 11.0.2+7)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.2+7, mixed mode)
```

**Expected behavior**
Expected to start stream and playing song.

**Screenshots/Stracktraces/Logs**
```
Connected successfully!
Skipping SecretBlock
Exception in thread "session-packet-receiver" java.nio.BufferUnderflowException
	at java.base/java.nio.Buffer.nextGetIndex(Buffer.java:641)
	at java.base/java.nio.HeapByteBuffer.get(HeapByteBuffer.java:162)
	at xyz.gianlu.librespot.core.Session$Receiver.run(Session.java:756)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Version/Commit**
I used version 0.6.2 obtained from https://github.com/librespot-org/librespot-java/releases

**Comment by pejobo** [comment]
Duplicate to #142

**Comment by devgianlu** [comment]
You can use `dev`. There'll soon be a release based on that branch.

**Comment by pejobo** [comment]
Yes, I build `dev` locally and can confirm it works. Tested on Ubuntu (open-jdk-11) and Arch Linux Arm (open-jdk-13).
A new release would be fine because as I understand the latest release is not usable for anyone.

**Comment by devgianlu** [comment]
I've just released [1.0.0](https://github.com/librespot-org/librespot-java/releases/tag/v1.0.0).
